src/languages/jp.json

```diff
@@ -20,24 +21,32 @@
                 "failed": "自動検出に失敗しました"
             }
         }
+    },
+        "windowTitle": {
+            "main": "SlimeTora: メイン",
+            "onboarding": "SlimeTora: セットアップウィザード",
+            "settings": "SlimeTora: {trackerName} の設定",
+            "pairing": "SlimeTora: ペアリング"
     },
```

Comment:
```suggestion
            }
        },
        "windowTitle": {
           "main": "SlimeTora: メイン",
           "onboarding": "SlimeTora: セットアップウィザード",
           "settings": "SlimeTora: {trackerName} の設定",
           "pairing": "SlimeTora: ペアリング"
        }
    },
```

Reply:
The window title's didn't work because "windowTitle" was not under the "main" key. It's supposed to be `main.windowTitle.X` not `windowTitle.X`